### ISSUE TITLE
Viral discovery workflow

### ISSUE THREAD

**Issue by AlfredDobradi** [AlfredDobradi/twitch-graph#3]
The workflow should be changed drastically. There should be no distinction between channels and users, even though they're separate entities in the API, in regards of discovering who follows who it shouldn't matter as users don't only follow active broadcasters.

To avoid confusion, the term "follower" means the person following the currently parsed user, the term "following" means the person or list of people followed by the currently parsed user.

To make later queries easier, users who meet a certain criteria - tbd - should be labeled as Broadcaster in Neo4j.

The new workflow should incorporate some kind of queuing system to keep track of users not parsed yet.
Additionally the script should always go by the users the currently parsed user follows. There are two reasons for this:
* It's a smaller set in average (popular broadcasters may have tens of thousands of followers, but only follow hundreds of channels which evens out the difference of smaller channels or non-broadcasters)
* To get the followers a separate API call is needed

To sum up the above, the new workflow should look like this:

* Parse initial user - either through CLI param or straight from queue
* Get following list and save to queue
* Iterate:
* * Get next person from queue
* * Parse user
* * Get followers and save to queue

Needless to say, a user shouldn't be parsed twice, no matter how many follow. This should be restricted in the queue not allowing duplicate entries.
